src/v/v8_engine/script.cc

```diff
@@ -92,11 +99,7 @@ void script::set_function(std::string_view name) {
     _function.Reset(_isolate.get(), function_val.As<v8::Function>());
 }
 
-// It is the first version for run. In next updates I use alient_thread for run
-// v8.
-void script::run(ss::temporary_buffer<char>& data) { run_internal(data); }
-
-void script::run_internal(ss::temporary_buffer<char>& data) {
+void script::run_internal(ss::temporary_buffer<char> data) {
```

Comment:
should this (and all methods that take code / data) use an iobuf? if these are run on an alien thread, are there things in iobuf or temporary buffer that might be problematic to execute?

Reply:
I need to construct ArrayBuffer for v8 on this obj. I think iobuf is not good for things like
```
auto store = v8::ArrayBuffer::NewBackingStore(
      data.get_write(), data.size(), v8::BackingStore::EmptyDeleter, nullptr);
```

Reply:
on the other hand, if a script is large and our memory is fragmented, then we'll crash.

Reply:
sorry, I do not clearly understand. Could you please add more detail. (What does script is large mean, What about data passing to v8)

Reply:
Add ticket ch3841